src/Components/Jumbotron.css

```diff
@@ -2,6 +2,10 @@
    padding: 7rem;
 }
 
+#reservationButton {
```

Comment:
This works just fine in this case, but in general, some advice: try to leverage css classes as much as possible, to keep from having to repeat certain things. For example, since you're defining some of these fonts multiple times for different elements, something that could make your work a bit easier would be to define a class of:
 `.button-font {
         font-family: 'Quattrocento', serif;
}`

From there you could just apply that to any element that needs that font! This makes your code and your workflow a bit more efficient, and is a good habit to build.